donut/modules/editor/helpers.py

```diff
@@ -100,7 +100,6 @@ def rename_title(old_filename, new_filename):
     remove_file_from_db(old_filename)
 
     if os.path.exists(old_path) and not os.path.exists(new_path):
-        os.rename(old_path, new_path)
```

Comment:
Why did this get removed? It's causing the tests to fail

Reply:
whoops removed it on accident during merging. I'll add it back